core/storage/user/gae_models.py

```diff
@@ -689,6 +689,80 @@ def export_data(cls, user_id):
         return user_data
 
 
+class LearnerGoalsModel(base_models.BaseModel):
+    """Keeps track of all the topics to learn.
+
+    Instances of this class are keyed by the user id.
+    """
+
+    # IDs of all the topics selected by the user to learn.
+    topic_ids_to_learn = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+    # IDs of all the topics selected by the user to master.
+    topic_ids_to_master = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+
+    @staticmethod
+    def get_deletion_policy():
+        """Model contains data to delete corresponding to a user: id field."""
+        return base_models.DELETION_POLICY.DELETE
+
+    @staticmethod
+    def get_model_association_to_user():
+        """Model is exported as one instance per user."""
+        return base_models.MODEL_ASSOCIATION_TO_USER.ONE_INSTANCE_PER_USER
+
+    @classmethod
+    def get_export_policy(cls):
+        """Model contains data to export corresponding to a user."""
+        return dict(super(cls, cls).get_export_policy(), **{
+            'topic_ids_to_learn': base_models.EXPORT_POLICY.EXPORTED,
+            'topic_ids_to_master': base_models.EXPORT_POLICY.NOT_APPLICABLE
```

Comment:
Why "not applicable"?

Reply:
Because we don't store the topic ids to master for now. I added this field to avoid migration in future.

Reply:
I think you can still export it, right? Worst case, it's just empty at the moment. But the data policies should be correct at the outset.

Reply:
Done.